public/lesswrong-editor/src/ckeditor5-footnote/README.md

```diff
@@ -0,0 +1,70 @@
+# CKEditor 5 Footnotes
+This plugin allows for inserting footnotes in CKEditor 5. This README is aimed at developers, to explain the overall structure of this plugin, giving an overview of the code and how it's organized, while also explaining some of the subtleties of CKEditor.
+
+## Background, CKEditor ecosystem
+### Information Structure
+Documents in CKEditor are stored as a tree structure, where each tree node has abstract class `Node`. All `Node`s are either `Element`s or `Text` nodes. All `Text` nodes are leaves in the tree, containing only characters. `Element`s can have child nodes, but don't have to. Both node types can be assigned attributes.
+
+### Positioning
+There are two integer measures of location for a given `Node` within its parent: index and offset. A `Node`'s index is the number of sibling nodes (`Element` or `Text`) that occur before it; a `Node`'s offset is the number of `Element` nodes and characters that occur before it. In other words, the only difference is that a `Text` node's `offsetSize` is its character count, while it's "index size" is always 1. An `Element` node's `offsetSize` is always 1. This is also to say that a node's children don't contribute to either the offset or index of the node's siblings.
```

Comment:
(Apparently I made my previous comment on this section on the commit itself so it's no showing up in the review. But anyway: )

I wonder if this is not important enough to mention here. Do we actually use this information?